src/features/envVisualizer/EnvVisualizer.tsx

```diff
@@ -9,15 +9,32 @@ type SetVis = (vis: React.ReactNode) => void;
 export default class EnvVisualizer {
   /** callback function to update the visualization state in the SideContentEnvVis component */
   private static setVis: SetVis;
+  private static printableMode: boolean = false;
+  private static context: Context;
+  public static togglePrintableMode(): void {
+    EnvVisualizer.printableMode = !EnvVisualizer.printableMode;
+  }
+  public static getPrintableMode(): boolean {
+    return EnvVisualizer.printableMode;
+  }
 
   /** SideContentEnvVis initializes this onMount with the callback function */
-  static init(setVis: SetVis) {
+  static init(setVis: SetVis, width: number, height: number) {
+    Layout.visibleHeight = Math.max(isNaN(height) ? window.innerHeight - 200 : height - 200, 200);
```

Comment:
can we assign names to these constants (e.g. 200, 50)? in general, we should avoid magic numbers/ values